packages/banner/src/__stories__/Banner.stories.js

```diff
@@ -2,15 +2,38 @@ import React from "react";
 
 import { storiesOf } from "@storybook/react";
 import { action } from "@storybook/addon-actions";
-import { text, select } from "@storybook/addon-knobs/react";
+import { text, select, boolean } from "@storybook/addon-knobs/react";
+import { makeSelectOptions } from "@hig/storybook/utils";
 
 import { Button } from "hig-react";
 import Banner from "../Banner";
 
+const typeOptions = makeSelectOptions(Banner.types);
+const placementOptions = makeSelectOptions(Banner.placements);
+
+const knobGroupIds = {
```

Comment:
:clap: I didn't know I needed this.

Reply:
I know! I didn't know this was a thing. Very cool.